.github/workflows/codeql.yml

```diff
@@ -7,6 +7,17 @@ on:
   pull_request:
     branches:
       - main
+    paths:
+      - 'source/**'
+      - 'clik/**'
+      - 'modules/**'
+      - 'examples/**'
+      - 'cmake/**'
+      - 'hal/**'
+      - '.github/actions/do_build_ock/**'
+      - '.github/actions/setup_build/**'
+      - '.github/workflows/run_pr_tests.yml'
+      - 'CMakeLists.txt'
```

Comment:
you need to add codeql.yml here and remove the other workflow

Reply:
Given what it does, I'm now wondering if there are any implications in altering the provided CodeQL scope in this way? Should I give Carlos a ping?

Reply:
Doesn't appear to be any hard and fast wisdom on best practice re. benefits/risks here.
I'll go ahead and add the codeql.yml that I missed to the list.

Reply:
not add, replace the other workflow. No need to ping.

Reply:
Done